iothub/device/src/Transport/Amqp/AmqpAuthenticationRefresher.cs

```diff
@@ -150,6 +140,17 @@ private async Task RefreshLoopAsync(DateTime refreshesOn, CancellationToken canc
                 }
             }
         }
+        private async Task<DateTime> SendTokenWithRetryAsync(CancellationToken cancellationToken)
+        {
+            var retryHandler = new RetryHandler(_retryPolicy);
```

Comment:
Why create a new retry handler? We have one created for the client already. Perhaps we should pass that down to this object, instead of the IRetryPolicy?